http-api/src/main/java/net/runelite/http/api/chat/ChatClient.java

```diff
@@ -30,8 +30,7 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
-import java.util.Collection;
-import java.util.Set;
+import java.util.HashMap;
```

Comment:
Why is this used over `java.util.Map`? I don't see anything which specifies a HashMap specifically is needed.

Reply:
This was only for consistency. Can change it